gatsby/content/docs/moderation-guide.mdx

```diff
@@ -190,12 +190,12 @@ If you're using Element, when you go to view more information about a particular
 ![Element-deactivate-btn](/docs/projects/images/moderation8.png)
 
 
-Removing abusive rooms is done by calling the /shutdown_room API, e.g:
+Removing abusive rooms is done by calling the delete room API, e.g:
 
 
 ```
-curl 'https://matrix.org/_synapse/admin/v1/shutdown_room/!roomid:matrix.org' \
--H "Authorization: Bearer $token" -XPOST \
+curl 'https://matrix.org/_synapse/admin/v2/rooms/!roomid:matrix.org' \
+-H "Authorization: Bearer $token" -X DELETE \
 -d'{"new_room_user_id": "@abuse:matrix.org", "message": "This room has been removed from the matrix.org homeserver due to violating the terms of use.", "room_name": "Room unavailable"}'
```

Comment:
@uranuspucksaxophone I think you need to add `block: true` here for this to be equivalent to the shutdown_room API. Otherwise you are just deleting it but not disallowing people from joining in the future.